test/integration/costumes.test.js

```diff
@@ -38,6 +38,32 @@ describe('Working with costumes', () => {
         await expect(logs).toEqual([]);
     });
 
+    test('Adding a costume by surprise button', async () => {
+        await loadUri(uri);
+        await clickXpath('//button[@title="tryit"]');
+        await clickText('Costumes');
+        const el = await findByXpath('//button[@aria-label="Choose a Costume"]');
+        await driver.actions().mouseMove(el)
+            .perform();
```

Comment:
Cool!  If we're doing this a lot already, maybe a `moveMouseToElement` (or something) helper?  I can imagine needing more of this in the future with drag and drop, for example.

Reply:
Yeah, I'm not sure what the right level of abstraction is for it though, because if you need to do anything else you'll need to drop back down to this level. I think we should wait for some more uses.